code/modules/mob/living/carbon/touch.dm

```diff
@@ -0,0 +1,7 @@
+/mob/living/carbon/proc/touch(var/datum/reagents/from, var/datum/reagents/target, var/amount = 1, var/multiplier = 1, var/copy = 0)
```

Comment:
This doesn't make sense being a human proc. If parent specific behaviour is necessary, then you'd most likely have to implement an on_reagent_touch, which the reagent then calls. So basically, the call stack would be:
`reagent/touch -> reagent/touch_specific -> specific/on_reagent_touch`. This would allow for children of reagent to override their behaviour, and for targets to do the same. Which should cover most of the cases.

Reply:
I just "cloned" what proc/ingest did as it does the same thing as proc/touch. ingest doesn't need parent specific behavior also. Both of them are procs because it makes the code nicer and more formatted, but I'll see what I can do.